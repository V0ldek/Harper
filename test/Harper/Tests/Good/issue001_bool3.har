main :: Bool;
main = true and false;