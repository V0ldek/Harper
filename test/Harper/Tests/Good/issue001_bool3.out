Execution ended with value: false